packages/vue3-vite/src/generators/library/schema.json

```diff
@@ -35,6 +35,12 @@
       "description": "Decide whether to use local or global path alias in tsconfig",
       "enum": ["local", "global"],
       "default": "global"
+    },
+    "test": {
```

Comment:
Add a `test` option in both library generator and app generator.
Both can be either `vitest` or `jest`, defaults to `vitest`.

Reply:
👍